rolling/distribution.yaml

```diff
@@ -5529,6 +5529,21 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_fastrtps.git
       version: rolling
     status: developed
+  rosmobile_build_tools:
+    doc:
+      type: git
+      url: https://github.com/Application-UI-UX/rosmobile_build_tools/blob/main/README.md.git
+      version: main
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: git@github.com:Application-UI-UX/rosmobile_build_tools.git
+      version: 0.4.2-1
+    source:
+      type: git
+      url: https://github.com/Application-UI-UX/rosmobile_build_tools/blob/main/README.md.git
```

Comment:
This must point to the base of the repository, and end in `.git`.  Thus this needs to be `https://github.com/Application-UI-UX/rosmobile_build_tools.git`.

Reply:
yea, my fault. I saw doc and source and though hmmm, give the README.md location, even tho other package I published(this package as well) did not need such things.